cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,43 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+YARN containers spawned by a CDAP application use the default container logback, ``logback-container.xml``,
+packaged with CDAP. This ``logback.xml`` does log rotation, and depending on the use case, may be sufficient.
+However, it is also possible to specify a custom ``logback.xml`` for a CDAP application.
+The packaged ``logback.xml`` is then used for each container spawned by a program of the application.
```

Comment:
The packaged ``logback.xml`` is then used for each container launched for the application.